lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
It's an optional parameter: either we use an implementation similar to the one above (but that raises questions about handling of non-optional parameters), or add support for conditional validators...